packages/style-engine/src/index.ts

```diff
@@ -91,3 +91,6 @@ export function getCSSRules(
 
 	return rules;
 }
+
+// Export style utils.
+export { getCSSVarFromStyleValue } from './styles/utils';
```

Comment:
This is now part of the style engine's public API.

Reply:
Given this is to be part of the public API, is there an opportunity to try and improve the naming here?